### PR TITLE
#3741: replaced Go Crazy text with Search

### PR DESCRIPTION
**Pull request by rajujha373**

### CODE REVIEW

File: readthedocs/templates/search/base_facet.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
           <input type="hidden" name="selected_facets" value="{{ selected_facets }}" id='id_selected_facets'>
         </div>
         <div class="submit-input-wrapper">
-          <input type="submit" value="{% trans "Go crazy." %}" id="id_search_button">
+          <input type="submit" value='{% trans "Search" %}' id="id_search_button">
```
Comment (stsewd) [inline readthedocs/templates/search/base_facet.html]: You shouldn't change the double quotes to single quotes here, html needs `"`.

File: readthedocs/templates/search/search.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
               </div>
               <div class="submit-input-wrapper">
                 {% comment %}Translators: This is about starting a search{% endcomment %}
-                <input type="submit" value="{% trans "Go crazy." %}">
+                <input type="submit" value='{% trans "Search" %}'
```
Comment (stsewd) [inline readthedocs/templates/search/search.html]: I think you removed the `>` accidentally here

File: readthedocs/templates/search/base_facet.html (hunk 1 above)
Comment (rajujha373) [reply readthedocs/templates/search/base_facet.html]: <code>"{% trans 'search' %}" </code> 
This syntax will do i guess. I think both shall not be double quotes.. Do correct me if i'm wrong

File: readthedocs/templates/search/search.html (hunk 2 above)
Comment (rajujha373) [reply readthedocs/templates/search/search.html]: oops.. my bad :D I'll correct it for sure :)

**Comment by agjohnson** [comment]
I'm glad someone was able jump in and clean this up, thanks for the contribution @rajujha373 ! 🎉

**Comment by RichardLitt** [comment]
Well done, @rajujha373! Welcome to the family. Make yourself at home. :)

**Pull request merged**